src/components/PlantsV2/components/HighestAndLowestMortalityRateSpeciesCard.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import OverviewItemCard from 'src/components/common/OverviewItemCard';
+import strings from 'src/strings';
+import { Box, Divider, Typography, useTheme } from '@mui/material';
+import { ObservationResults } from 'src/types/Observations';
+import { selectSpecies } from 'src/redux/features/species/speciesSelectors';
+import { useAppSelector } from 'src/redux/store';
+
+type HighestAndLowestMortalityRateSpeciesCardProps = {
+  observation?: ObservationResults;
+};
+
+export default function HighestAndLowestMortalityRateSpeciesCard({
+  observation,
+}: HighestAndLowestMortalityRateSpeciesCardProps): JSX.Element {
+  const theme = useTheme();
+  const species = useAppSelector(selectSpecies);
+
+  let highestMortalityRate = 0;
+  let highestSpecies = '';
+
+  observation?.species.forEach((sp) => {
+    if (sp.mortalityRate && sp.mortalityRate > highestMortalityRate) {
+      highestMortalityRate = sp.mortalityRate;
+      // highestSpecies = sp.speciesName || '';
+      highestSpecies = species?.find((iSpecies) => iSpecies.id === sp.speciesId)?.scientificName || '';
+    }
+  });
+
+  let lowestMortalityRate = 100;
+  let lowestSpecies = '';
+
+  observation?.species.forEach((sp) => {
+    if (sp.mortalityRate && sp.mortalityRate < lowestMortalityRate) {
```

Comment:
if `sp.mortalityRate` is '0', the comparison will be ignored.. we should probably look for `!isNaN(mortalityRate)`